okhttp/src/main/java/okhttp3/ResponseBody.java

```diff
@@ -103,7 +103,10 @@ public final Reader charStream() {
   /**
    * Returns the response as a string decoded with the charset of the Content-Type header. If that
    * header is either absent or lacks a charset, this will attempt to decode the response body as
-   * UTF-8.
+   * UTF-8. Closes {@linkplain ResponseBody} automatically.
```

Comment:
Can you please explain when you prefer `@link` and when `@linkplain`?

According to http://www.tutorialspoint.com/java/java_documentation.htm it looks like style preference rather than functional, right?